tests/utils/verification/dockercontainer.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This util provides various helper methods that can be used by different tests to
+// fetch information related to docker service and containers.
+
+package verification
+
+import (
+	"log"
+	"strings"
+
+	"strconv"
+
+	"github.com/vmware/docker-volume-vsphere/tests/constants/dockercli"
+	util "github.com/vmware/docker-volume-vsphere/tests/utils/dockercli"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/misc"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/ssh"
+)
+
+// GetContainerName returns full container name based on given short name
+func GetContainerName(hostName, shortName string) (string, error) {
```

Comment:
`GetContainerName ` & `GetAllContainers` should move out from verifier package ... both of them spoils the context of `verifier` .. /util/dockercli would be a better home.